src/engines/arps/ARP/audio.cpp

```diff
@@ -41,6 +45,7 @@ namespace otto::engines::arp {
     //Resets state. It is necessary to do this AFTER clearing the midi data, 
     //otherwise we miss note-off events.
     if (stop_flag){
+      services::ClockManager::current().stop(true);
```

Comment:
When we get the sequencer as well, im thinking you pass a `ClockSource` enum to `start()` and `stop()`, which will decide which source gets to control the clock. So the sequencer will have a higher priority than `ClockSource::arpeggiator`

Reply:
Yes, I agree. This is a temporary fix until we get that whole thing.